docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sparse/embedding-bag-offsets-15.rst

```diff
@@ -0,0 +1,154 @@
+        ), "Attribute per_sample_weights is only supported in sum reduction."
+        if per_sample_weights is None:
+            per_sample_weights = np.ones_like(indices)
+        embeddings = []
+        for emb_idx, emb_weight in zip(indices, per_sample_weights):
+            embeddings.append(emb_table[emb_idx] * emb_weight)
+        previous_offset = offsets[0]
+        bags = []
+        offsets = np.append(offsets, len(indices))
+        for bag_offset in offsets[1:]:
+            bag_size = bag_offset - previous_offset
+            if bag_size != 0:
+                embedding_bag = embeddings[previous_offset:bag_offset]
+                reduced_bag = np.add.reduce(embedding_bag)
+                if reduction == "mean":
+                    reduced_bag = reduced_bag / bag_size
+                bags.append(reduced_bag)
+            else:
+                # Empty bag case
+                if default_index is not None and default_index != -1:
+                    bags.append(emb_table[default_index])
+                else:
+                    bags.append(np.zeros(emb_table.shape[1:]))
+            previous_offset = bag_offset
+        return np.stack(bags, axis=0)
+
+
+**Attributes**:
+
+* *reduction*
+
+  * **Description**: reduction mode. 
+  * **Range of values**:
+
+    * sum - compute weighted sum, using corresponding values of ``per_sample_weights`` as weights if provided.
+    * mean - compute average of values in bag. Input ``per_sample_weights`` is not supported and will raise exception.
+
+  * **Type**: ``string``
+  * **Default value**: sum
+  * **Required**: *no*
+
+**Inputs**:
+
+* **1**: ``emb_table`` tensor containing the embedding lookup table of the module of shape ``[num_emb, emb_dim1, emb_dim2, ...]`` and  of type *T*. **Required.**
+* **2**: ``indices`` tensor of shape ``[num_indices]`` and of type *T_IND*. **Required.**
+* **3**: ``offsets`` tensor of shape ``[batch]`` and of type *T_IND* containing the starting index positions of each "bag" in ``indices``. Maximum value of offsets cannot be greater than length of ``indices``. **Required.**
+* **4**: ``default_index`` scalar of type *T_IND* containing default index in embedding table to fill empty "bags". If set to ``-1`` or not provided, empty "bags" are filled with zeros. Reverse indexing using negative values is not supported. **Optional.**
+* **5**: ``per_sample_weights`` tensor of the same shape as ``indices`` and of type *T*. Supported only when *reduction* attribute is set to ``"sum"``. Each value in this tensor are multiplied with each value pooled from embedding table for each index. Optional, default is tensor of ones. **Optional.**
+
+**Outputs**:
+
+* **1**: tensor of shape ``[batch, emb_dim1, emb_dim2, ...]`` and of type *T* containing embeddings for each bag.
+
+**Types**
+
+* *T*: any numeric type.
+* *T_IND*: ``int32`` or ``int64``.
+
+**Example**
+
+.. code-block:: xml
+
+   <layer ... type="EmbeddingBagOffsets" ... >
+       <input>
```

Comment:
Usually every example has a title, it should be added to emphasise what is presented.
Please take a look at any other spec for reference, for instance: 
https://github.com/openvinotoolkit/openvino/blob/55e3bee5fd20ab726595bba6c4040c16749fe6b5/docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sequence/scaled-dot-product-attention.rst?plain=1#L95-L100

Also would be great to add an example of EmbeddingBagOffsets with new reduction attribute and "mean" value.

Reply:
Done, + adjusted per_sample_weights to better show difference between sum and mean